.travis.yml

```diff
@@ -1,8 +1,10 @@
-sudo: false
+sudo: required
```

Comment:
Interesting, this should be running in Sauce Labs, so you shouldn't need this. I'd prefer to not have to do this and fix the sauce issue.